version.pri

```diff
@@ -1,8 +1,9 @@
 VERSION_MAJOR = 1
 VERSION_MINOR = 4
 VERSION_FIX = 0
-VERSION_RC = 99
-VERSION_SUFFIX = ''
+# for RC (v1.2.3-rc4) or dev (v1.2.3-dev456)
+VERSION_NUMBER = 999
```

Comment:
this has been increased to 3 digits, in case we have more than 99 commits on the dev (nightlies) build.
I guess it won't cause any trouble to the current build, since the global number will have 1 more digit (hence bigger).
Or is there a maximum value for this number (and I would be above now)
@mbernasocchi can you confirm?

Reply:
the main issue is that ve are getting one order of magnitude closer to the max version size.
https://stackoverflow.com/questions/24246143/maximum-length-of-android-versionname-versioncode-manifest